doc/concepts/configuration/configuration_connections.rst

```diff
@@ -0,0 +1,292 @@
+    :language: yaml
+    :dedent:
+
+
+
+
+.. _configuration_connections_advertise:
+
+Advertise settings
+------------------
+
+Advertise settings (:ref:`iproto.advertise.* <configuration_reference_iproto_advertise>`) let other cluster members or clients know how to connect to the current Tarantool instance.
+These settings might include the credentials required to connect to this instance, a URI used to listen for incoming requests, and SSL settings.
+
+If an advertise URI is not specified explicitly, a :ref:`listen URI <configuration_connections_listen_uri>` of this instance is used.
+In this case, you need at least to specify credentials for connecting to this instance.
+
+..  NOTE::
+
+    If several listen URIs are specified, the first one is used as an advertise URI.
+
+.. _configuration_connections_advertise_credentials:
+
+Connection credentials
+~~~~~~~~~~~~~~~~~~~~~~
+
+In the example below, the ``iproto.advertise.peer`` option is used to inform other replica set members that the ``replicator`` user should be used to connect to the current instance:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/advertise_peer/config.yaml
+    :language: yaml
+    :start-after: roles: [replication]
+    :end-at: login: replicator
+    :dedent:
+
+In a sharded cluster, ``iproto.advertise.sharding`` specifies that a router and rebalancer should use the ``storage`` user to connect to storages:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/config.yaml
+    :language: yaml
+    :start-at: iproto
+    :end-at: login: storage
+    :dedent:
+
+
+.. _configuration_connections_advertise_explicitly:
+
+Advertise URI
+~~~~~~~~~~~~~
+
+If required, you can specify an advertise URI explicitly by setting up the :ref:`iproto_advertise.\<peer_or_sharding\>.uri <configuration_reference_iproto_advertise.peer_sharding.uri>` option.
+In the example below, ``iproto.listen`` includes two URIs that can be used to connect to ``instance001`` but only the second one is used to advertise this instance to other replica set peers:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/advertise_peer/config.yaml
+    :language: yaml
+    :start-at: instance001:
+    :end-before: instance002:
+    :dedent:
+
+The ``iproto_advertise.<peer_or_sharding>.uri`` option can also accept an FQDN instead of an IP address:
+
+..  code-block:: yaml
+
+    instance001:
+      iproto:
+        listen:
+        - uri: '192.168.0.101:3301'
+        advertise:
+          peer:
+            uri: 'server001.example.com:3301'
+
+
+.. _configuration_connections_advertise_uri_ssl:
+
+SSL parameters
+~~~~~~~~~~~~~~
+
+:ref:`SSL parameters <configuration_connections_ssl>` for an advertise URI should be set only if this :ref:`advertise URI is specified explicitly <configuration_connections_advertise_explicitly>`.
+Otherwise, SSL parameters of a listen URI are used and no additional configuration is required.
```

Comment:
These sentences are a bit complicated due to several negations. AFAIU, you want to say that if `uri` or `params` are set in an advertise URI, `login`, `password`, `uri`, `params` are not borrowed from the listen URI. Is it correct?

Reply:
Not exactly. Here I want to emphasise that if `iproto_advertise.<peer_or_sharding>.uri` is specified manually, then SSL settings can't be taken from `iproto.listen` and should be specified manually.